tools/build-launchui.js

```diff
@@ -28,20 +28,22 @@ function buildLaunchUI() {
   const nodeDir = path.join( __dirname, '../deps/node' );
   const libuiDir = path.join( __dirname, '../deps/libui' );
 
-  const nodeSrc = path.join( nodeDir, process.platform == 'darwin' ? 'out/Release/libnode.57.dylib' : 'out/Release/lib.target/libnode.so.57' );
```

Comment:
On linux here now 64 for NodeJS, not 57. To avoid such dependency, I added reliable way to find libnode file. Just scan dir and find by mask.

Reply:
Fixed